test/energy_scan/mult_and_spectra_plotter.py

```diff
@@ -336,17 +346,20 @@ def is_number(s):
             # print data
 
 def plotting(data1, data2, config_file, smash_code_version, output_folder):
+    plot_counter=0
     quantities = data1.quantities.union(data2.quantities)
     pdglist = data1.pdglist.union(data2.pdglist)
     pdglist_abs = np.unique(np.abs(np.array(list(pdglist))))
     colliding_systems = data1.colliding_systems.union(data2.colliding_systems)
-    colliding_systems_list = list(colliding_systems)
+    if('afterburner' in colliding_systems):
+        colliding_systems={'afterburner'}
     energies = sorted(list(data1.energies.union(data2.energies)))
-
     for quantity in quantities:
         if ('spectra' in quantity): continue
+        #if ('afterburner' in colliding_systems): continue
```

Comment:
If not needed anymore, perhaps it can be canceled instead of just being commented out.

Reply:
Thank you, I forgot to remove a few stray comments.